app/controllers/devise_sessions_controller.rb

```diff
@@ -64,7 +89,12 @@ def phone_verify
     end
   end
 
-  def phone_resend; end
+  def phone_resend
+    return if request.get?
```

Comment:
I feel like if we have method-specific behaviour, it should be done in the routes.  It's not a strong feeling though so feel free to leave it if you think this is better.